### DISCUSSION TITLE
Could not load database driver: MssqlEngineSpec

### DISCUSSION THREAD

**Discussion by lx1054331851** [apache/superset#15022] [Q&A / Help]
I install superset from docker hub, and install the pymssql in the docker superset container, but I still cannot connect to the database, the error is :
Could not load database driver: MssqlEngineSpec
2021-06-07 15:17:17,452:WARNING:superset.views.base:Could not load database driver: MssqlEngineSpec

**Comment by RockNHawk** [comment]
I have the same issue

**Comment by RockNHawk** [comment]
run pip install pymssql with root user,

then use this connectiong string resolved my issue:
mssql+pymssql://<Username>:<Password>@<Host>:<Port-default:1433>/<Database Name>/?Encrypt=yes
https://superset.apache.org/docs/databases/sql-server
